Snakefile

```diff
@@ -20,6 +20,8 @@ def download_file_from_url(url, save_to):
 
 # Configuration constants
 L_FREQS = (None, 1)
+ICA_N_COMPONENTS = 0.999
```

Comment:
Can you add a comment here and explain why a number of components for ICA is not an integer number?

Reply:
Done.